modules/src/ics04_channel/channel.rs

```diff
@@ -142,20 +133,16 @@ impl Counterparty {
                 .map_err(|e| Kind::IdentifierError.context(e))?,
         })
     }
-}
-
-impl ChannelCounterparty for Counterparty {
-    type ValidationError = Error;
 
-    fn port_id(&self) -> String {
-        self.port_id.as_str().into()
+    pub fn channel_id(&self) -> String {
+        self.channel_id.as_str().into()
     }
 
-    fn channel_id(&self) -> String {
-        self.channel_id.as_str().into()
+    pub fn port_id(&self) -> String {
+        self.port_id.as_str().into()
     }
```

Comment:
Any reason for changing the order of these functions?

Reply:
Mostly for organization and aesthetics. Due to the fact that this implementation block had only three methods, they were arranged in ascending natural order.